libc/src/__support/FPUtil/NearestIntegerOperations.h

```diff
@@ -141,7 +141,7 @@ LIBC_INLINE T round(T x) {
 }
 
 template <typename T, cpp::enable_if_t<cpp::is_floating_point_v<T>, int> = 0>
-LIBC_INLINE T round_using_current_rounding_mode(T x) {
+LIBC_INLINE constexpr T round_using_specific_rounding_mode(T x, int rnd) {
```

Comment:
Since you are here, would you mind updating the `enable_if` to be on the returning type instead:
```
template <typename T>
LIBC_INLINE constexpr typename cpp::enable_if_t<..., T> rounding_using_specific_rounding_mode() { ... }
```
Same for the other 2 functions below.

Reply:
CLion Nova warns about the `typename` keyword before `cpp::enable_if_t` being redundant.

Reply:
You can remove `typename` before `cpp::enable_if_t` if it's not needed.